### DISCUSSION TITLE
use random node.js debug port

### DISCUSSION THREAD

**Discussion by paulboudewijn** [directus/directus#9801] [Feature Requests]
When starting a directus development server with the default port (8080) already in use, vitejs automatically tries to find another one.
But this currently doesn't apply to the nodejs inspector, whenever it's default port (9229) is already in use debugging simply isn't possible.

Will using a random portno for the nodejs inspector break stuff? If not, I've got a PR ready.

**Comment by rijkvanzanten** [comment]
No clue! We don't have anything hardcoded to that particular node debugger port, so it should be fine to randomize it / make it configurable. The only thing I can think of is that the individual configurations of developer's IDEs might be hardcoded to that specific port 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
